cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
+  beforeEach(() => {
+    cy.loadPage('/streamed-epub');
+  });
+
+  it('table of content link should update the page content', () => {
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
+        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
+      })
+      .should('exist');
+
+    // Open TOC menu
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+
+    // Open chapter 1
+    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.wait(100); // wait for the nested iframe to render
+
+    cy.iframe(IFRAME_SELECTOR)
+      .find('.subtitle')
+      .contains('Down the Rab­bit-Hole');
+  });
+
+  it('next and previous button', () => {
+    // load iframe
+    cy.iframe(IFRAME_SELECTOR);
+
+    cy.findByRole('button', { name: '>' }).click();
+
+    cy.wait(100);
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
```

Comment:
I would also use this strategy to make a custom command to get the iframe: https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/#custom-command

Reply:
I was actually using [this](https://gitlab.com/kgroat/cypress-iframe) library to get the iframe element, but I think I can remove it and create the custom command from the blog since our use case is simple.